android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -5,7 +5,7 @@ connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
 connection.project.dir=
 eclipse.preferences.version=1
 gradle.user.home=
-java.home=/Library/Java/JavaVirtualMachines/temurin-17.jdk/Contents/Home
+java.home=/opt/homebrew/opt/sdkman/libexec/candidates/java/17.0.1-tem
```

Comment:
I will remove this change, but I think this file probably shouldn't be checked in to source control overall.